GameFiles/_integration.gsc

```diff
@@ -907,8 +989,8 @@ NoClipImpl()
     self SetClientDvar( "cg_thirdperson", 1 );
     self SetClientDvar( "sv_cheats", 0 );
 
-    self call [[level.overrideMethods["god"]]]( true );
-    self call [[level.overrideMethods["noclip"]]]( true );
+    self [[level.overrideMethods["god"]]]( true );
+    self [[level.overrideMethods["noclip"]]]( true );
```

Comment:
I don't think we can just remove the `call` keyword here. This was added because on IW4 it won't execute our "function pointers" without the call keyword.

Reply:
Do not remove the word call on IW engine. Calls to builtin will not work